Fabric.Authorization.API/scripts/Install-Authorization-Windows.ps1

```diff
@@ -709,4 +766,12 @@ if ($adminAccount) {Add-InstallationSetting "authorization" "adminAccount" "$adm
 
 Invoke-MonitorShallow "$authorizationServiceUrl"
 
+Write-Host "Upgrading all the users with an 'EDW Admin' role to also have the dosadmin Fabric.Auth role"	+Read-Host -Prompt "Installation complete, press Enter to exit"
```

Comment:
Do we need to remove the `Read-Host` here?

Reply:
oh, i guess this was something that got thrown on that line.  I will remove it.  It is a duplicate.